api/capi/include/nntrainer.h

```diff
@@ -65,6 +65,18 @@ typedef enum {
   ML_LAYER_TYPE_UNKNOWN    /**< Unknown Lyaer */
 } ml_layer_type_e;
 
+/**
+ * @brief Enumeration for the neural network summary verbosity of NNTrainer
+ * @since_tizen 6.x
+ */
+typedef enum {
+  ML_TRAIN_SUMMARY_MODEL = 0, /**< Overview of model
+                                   summary with one-line layer information*/
+  ML_TRAIN_SUMMARY_LAYER, /**< Detailed model summary with layer properties */
+  ML_TRAIN_SUMMARY_TENSOR /**< Model summary layer's including weight
+                             information */
+} ml_train_summary_verbose_type_e;
```

Comment:
ml_train_summary_type_e? (the typedef name is too verbose in IMO :))

Reply:
yes it self is verbose 🤣 Changed.